bin/check-issue-has-release.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+/* eslint-disable sitekit/acronym-case */
+/* eslint-disable no-console */
+
+function graphQlFetch(query, variables) {
+  return fetch('https://api.zenhub.com/public/graphql', {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${process.env.ZENHUB_GQL_API_TOKEN}`,
+      },
+      body: new Blob([JSON.stringify({ query, variables })], {
+        type: 'application/json',
+      }),
+    }).then((res) => res.json());
+}
```

Comment:
We can use `await` here, which is just a bit nicer/more idiomatic.

```suggestion
async function graphQlFetch(query, variables) {
  const response = fetch('https://api.zenhub.com/public/graphql', {
    method: 'POST',
    headers: {
      Authorization: `Bearer ${process.env.ZENHUB_GQL_API_TOKEN}`,
    },
    body: new Blob([JSON.stringify({ query, variables })], {
      type: 'application/json',
    }),
  });
    
  if (!response.ok) {
    throw new Error( 'ZenHub API request failed.' );
  }
  
  return response.json();
}
```